src/js/components/charts/BarChart.js

```diff
@@ -71,7 +72,7 @@ var BarChart = React.createClass({
       stack: this.getStack(),
       xScale,
       yScale,
-      clipPathID: _.uniqueId('clip')
+      clipPathID: `clip-${Math.random().toString().slice(2)}`
```

Comment:
can we create a util for this? all it would have to do is start with an id that is 1 and then everytime it gets called, the just increments it so it will always be unique. rather than doing this